src/pygrambank/api.py

```diff
@@ -21,7 +21,7 @@ def sheets_dir(self):
 
     def iter_sheets(self):
         for p in sorted(self.sheets_dir.iterdir(), key=lambda i: i.stem):
-            if p.is_file() and p.name != '.gitattributes':
+            if p.is_file() and len(p.name) >= 4 and p.name[-4:] == '.tsv':
```

Comment:
> if p.is_file() and p.suffix == '.tsv':

..should work.

Reply:
Sure thing

Reply:
Just got to figure out how to "approve" this in the UI.